### ISSUE TITLE
Safe META tag output in wp_head?

### ISSUE THREAD

**Issue by paulcanning** [WordPress-Coding-Standards/WordPress-Coding-Standards#1137]
My theme functions.php adds a meta tag to wp_head to output a viewport tag.

```
function tameside_add_viewport_meta_tag() {
	$output = '<meta name="viewport" content="width=device-width, initial-scale=1">';
	echo $output;
}
```

PHPCS warns that the output isn't escaped. I tried `esc_html_e()` but that spat the tag out as a string in the body.

I also tried `wp_kses()` but then the tag didn't appear at all.

What is the escape function I should use in this case?

**Comment by jrfnl** [comment]
@paulcanning Why are you using two statements for this ?
The output escaping sniff kicks in because you are echoing out an unescaped variable, but in this case, I don't see why you need the variable at all.... The below will not trigger the sniff.

```php
function tameside_add_viewport_meta_tag() {
	echo '<meta name="viewport" content="width=device-width, initial-scale=1">';
}
```

**Comment by paulcanning** [comment]
No reason, but fair enough!
